CONTRIBUTING_EXTERNAL.md

```diff
@@ -0,0 +1,131 @@
+
+Please open an issue of type [Feature request](https://github.com/HHS/grants-equity/issues/new?assignees=octocat&labels=enhancement&projects=&template=feature_request.yml&title=%5BFeature+Request%5D%3A+):
+
+In this issue, please describe the use case for the feature you would like to see -, what you need, why you need it, and how it should work. Team members will respond to the Feature request as soon as possible. Often, Feature request suggestions undergo a collaborative discussion with the community to help refine the need for the feature and how it can be implemented.
+
+### Non-Technical Contributions
+
+#### Documentation
+
+To contribute to documentation you find in this repository, feel free to use the GitHub user interface to submit a pull request for your changes. Find more information about using the GitHub user interface for PRs here.
+
+### Contribute to community discussions
+
+> 🚧 Tools and expanding avenues for community engagement are coming soon.
+
+### Sharing your story 
+
+Sharing how you or your organization have used the Grants Equity project is an important way for us to raise awareness about the project and its impact. Please tell us your story by sending us an email at `grants-equity@hhs.gov`.
+
+## Code Contributions
+
+The following guidelines are for code contributions. Please see [CONTRIBUTING.md](./CONTRIBUTING.md) for more information about the software development lifecylce on the project.
+
+### Getting Started
+
+This project is monorepo with several apps. Please see the [api](./api/README.md) and [frontend](./frontend/README.md) READMEs for information on spinning up those projects locally. Also see the project [documentation](./documentation) for more info.
+
+### Workflow and Branching
+
+This project follows [trunk-based development](./CONTRIBUTING.md#branching-model), so all contributions are directed toward the `main` branch.
+
+1.  Fork the project
+1.  Check out the `main` branch
+1.  Create a feature branch
+1.  Write code and tests for your change
+1.  From your branch, make a pull request against `hhs/grants-equity/main`
+1.  Work with repo maintainers to get your change reviewed
+1.  Wait for your change to be pulled into `hhs/grants-equity/main`
+1.  Delete your feature branch
+
+### Testing, Coding Style and Linters
+
+Each application has its own testing and linters. Every commit is tested to adhere to tests and the linting guidelines. It is recommended to run tests and linters locally before committing.
+
+### Issues
+
+External contributors should use the *Bug Report* or *Feature Request* [issue templates](https://github.com/HHS/grants-equity/issues/new/choose).
+
+### Pull Requests
+
+Pull requests should follow the conventions in [CONTRIBUTING.md](./CONTRIBUTING.md) with the following changes:
+
+1. Pull requests should be titled with `[Issue N] Description`. However if there is no issue, use `[External]` format.
```

Comment:
Would it be helpful for external PRs to include a description in the title, ie. `[Exteneral] Description`, instead of just `[External]`?

Reply:
I made this change, thanks.